docs/content/release_notes/whats_new.md

```diff
@@ -1,57 +1,13 @@
-# What's new in 1.11.0
+# What's new in 1.11.1
 
-IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detailed in the following section.
+IBM® block storage CSI driver 1.11.1 introduces the enhancements that are detailed in the following section.
 
-**General availability date:** 23 January 2023
+**General availability date:** 8 May 2023
 
-## New dynamic volume group support
+## Supported orchestration platforms for deployment
 
-The new volume group support allows dynamic management of the content of the groups. This feature is used by policy-based replication, introduced IBM Spectrum Virtualize 8.5.2 release.
```

Comment:
I think there is a lot of important information that cannot be deleted entirely as it is not shown in any other file.
I suggest we move it under another section or file and not delete this information entirely. but I'm not sure where you should move it